.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
```

Comment:
@ipchelsea You can add comment per line.
You can also refer to https://hwsc-org.github.io/wikis/fundamentals/code-review.html for further guideline.